jwst/resample/resample.py

```diff
@@ -410,6 +418,11 @@ def resample_variance_array(self, name, output_model, iscale):
                 model.meta.wcs.bounding_box
             )
 
+            try:
+                iscale = model.meta.iscale
+            except AttributeError:
+                iscale = 1.0
```

Comment:
```suggestion
            iscale = model.meta.iscale
```

We do need the `iscale` and so we expect models to have this attribute. It is better to let the code crash (and let the caller know that something is not right => we need to fix the code) than to quietly default to a potentially incorrect value for `iscale`.